spec/users_controller_spec.rb

```diff
@@ -0,0 +1,111 @@
+require_relative 'spec_helper'
+
+def app
+  UsersController.new
+end
+
+describe UsersController do
+  before do
+    @users = DB[:users]
+  end
+
+  after do
+    @users.truncate
```

Comment:
I couldn't get DB.transaction to work. Instead, I used an after block to truncate the table.